config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
+  get 'foods/new'
+  get 'foods/create'
```

Comment:
Having created resources for `foods` It would be better to discard this in order to adhere to `DRY` practices to make your code neat.